### DISCUSSION TITLE
How to do layers

### DISCUSSION THREAD

**Discussion by utiq** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#560] [Q&A] [answered]
How can I do layers? like in https://github.com/pixelmatix/SmartMatrix
As you can see in their demo, one can create layers, how could I do it with this library?

![SmartMatrix demo](https://github.com/pixelmatix/SmartMatrix/wiki/photos/examples.gif)

**Comment by Bestelus** [comment]
I'm also interested by using layers if it's possible, but there's no exemples for this.
Anyone knows how to do it ?

**Reply by mrcodetastic** [reply, marked as answer]
Have added this using an existing Library I wrote.

https://github.com/mrcodetastic/ESP32-HUB75-MatrixPanel-DMA/tree/master/examples/ScrollingTextLayer

**Comment by mrcodetastic** [comment]
Graphics stuff isn't really the purpose of this library.

But refer to my GFX_ Lite repo for a library to help with this.
